src/components/extras/MapContainer.vue

```diff
@@ -74,12 +95,38 @@ export default {
             );
 
             // Create marker
-            this.marker = new mapboxgl.Marker({
-                draggable: this.drag,
-            })
-                .setLngLat(this.mapConfig.center)
-                .setPopup(popup)
-                .addTo(this.map);
+            if (this.spotsList) {
+                this.marker = new mapboxgl.Marker({
+                    draggable: this.drag,
+                })
+                    .setLngLat(this.mapConfig.center)
+                    .setPopup(popup)
+                    .addTo(this.map);
+            }
+
+            // If spot details is center
```

Comment:
is this comment valid?
spotDetails name does suggest that it should be set for center.